RemotingSample/Packages/packages-lock.json

```diff
@@ -77,7 +77,7 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.inputsystem": {
-      "version": "1.3.0",
```

Comment:
not sure if these changes can be made

Reply:
These should be okay! They're mainly dependency updates from our adoption of the 1.5.3 version of Unity's OpenXR package